app/com/linkedin/drelephant/spark/heuristics/ConfigurationHeuristic.scala

```diff
@@ -148,6 +158,20 @@ object ConfigurationHeuristic {
       case Some(_) => DEFAULT_SERIALIZER_IF_NON_NULL_SEVERITY_IF_RECOMMENDATION_UNMET
     }
 
+    //The following thresholds are for checking if the memory and cores values (executor and driver) are above normal. These thresholds are experimental, and may change in the future.
+    val DEFAULT_SPARK_MEMORY_THRESHOLDS =
```

Comment:
They are the same now, but we may want to separate driver and executor thresholds -- there is just 1 driver, so it is not as problematical if users allocate more memory/cores for the driver, so we could set the driver threshold higher. This could also be done later.

Reply:
Yaa.. that makes sense. Sure!